e2etests/testcase_test.go

```diff
@@ -3382,4 +3382,64 @@ var testCases = []testCase{
 			},
 		},
 	},
+	{
+		image:                   "ubi9/ubi:9.0.0-1576@sha256:f22a438f4967419bd477c648d25ed0627f54b81931d0143dc45801c8356bd379",
```

Comment:
can we add a check for some executables as well to ensure `--whatprovides` still works as expected?

Reply:
:+1: